### DISCUSSION TITLE
Bot creation

### DISCUSSION THREAD

**Discussion by MrFofanah** [community/community#120139] [Programming Help]
How to create a WhatsApp bot

**Comment by ebndev** [comment]
Thanks for posting in the GitHub Community, @MrFofanah! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction. Let us know what kind of bot you want to build!
